content/en/docs/troubleshooting.md

```diff
@@ -1,6 +1,88 @@
 ---
 title: "Troubleshooting"
-description: "Troubleshooting common problems with Notary"
+description: "Troubleshooting common problems with Notation"
 type: docs
 weight: 7
----
\ No newline at end of file
+---
+
+> NOTE:
+>
+> Suggest a troubleshooting item by raising a PR to our [documentation repository](https://github.com/notaryproject/notaryproject.dev)
+
+## Enabling notation CLI commands logging
+
+Printing log messages for debugging the problems can be enabled using the `--debug` or `-d` flag. Using the `--verbose` or `-v` flag to set verbosity level will be enough for most cases.
+
+For example:
+
+```shell
+# Enable the debugging logs
+notation sign -d localhost:5000/net-monitor@sha256:sha256:xxx
+
+# Enable the verbose logs
+notation verify -v localhost:5000/net-monitor@sha256:sha256:xxx
+```
+
+## When I verify an artifact, I get the error 'Error: open $HOME/.config/notation/trustpolicy.json: no such file or directory'
+
+Users need to configure trust policies before verifying artifacts. The trust policies are configured in a file named `trustpolicy.json`, which is stored under notation configuration directory.
+
+For Linux, the notation configuration directory is `${HOME}/.config/notation/`. For macOS, the notation configuration directory is `${HOME}/Library/Application Support/notation/`. For Windows, the notation configuration folder is `%AppData%\Roaming\notation\`.
+
+Using CLI command to configure trust policies is planned in Notation future release. Before that, you can refer to this [guide](https://github.com/notaryproject/notation/blob/v1.0.0-rc.1/specs/commandline/verify.md#configure-trust-policy) to configure trust policies manually.
+
+## When I verify an artifact, I get the error 'ERRO authenticity validation failed. Failure reason: error while loading the trust store, "$HOME/.config/notation/truststore/x509/ca/mytruststore" does not exist'
```

Comment:
@yizha1  Can you shorten each heading and move the detailed problem description below it? It causes the heading and right sidebar layout to look not pretty on the [preview page](https://deploy-preview-125--notarydev.netlify.app/docs/troubleshooting/). See [Prometheus FAQ documentation](https://prometheus.io/docs/introduction/faq/) as a good sample.

![image](https://user-images.githubusercontent.com/40452856/219572078-58aba5ce-b076-4696-b134-930ebaf41e17.png)